### ISSUE TITLE
Queue items

### ISSUE THREAD

**Issue by fishcracker** [boxeehacks/boxeehack#202]
I apologize if I am doing this wrong, I am new to Github.  Is there a way to request a queuing similar to the XBMC? Im suprised boxee didn't have a better queue setup for whole seasons instead of an individual show. Thank you.

**Comment by FabianMeul** [comment]
Auto-playing the next episode is a feature I'm craving for.

**Comment by Silent337** [comment]
Double + 1 to both :)

**Comment by kelmk** [comment]
I would remove the watch later queue and replace it with a new files queue. As new shows are movies are scanned they get added to the list until they are watched.

**Comment by fishcracker** [comment]
I would gladly lose the watch later queue hands down.  Even an auto play for video would be awesome. They have it for music. Wish I knew how to do any of this stuff!

**Comment by Mike120** [comment]
Auto play or a queue would be great!

**Comment by airjer** [comment]
Yes! I never understood why this hasn't been a default feature. Auto play next or a queue set up.
